rye/src/cli/shim.rs

```diff
@@ -101,6 +114,51 @@ fn get_shim_target(target: &str, args: &[OsString]) -> Result<Option<Vec<OsStrin
         if target == "pip" || target == "pip3" {
             return Ok(Some(get_pip_shim(&pyproject, args, CommandOutput::Normal)?));
         }
+
+    // Global shims (either implicit or requested)
+    } else if target == "python" || target == "python3" {
+        let config = Config::current();
+        let mut remove1 = false;
+
+        let version_request = if let Some(rest) = args
+            .get(1)
+            .and_then(|x| x.as_os_str().to_str())
+            .and_then(|x| x.strip_prefix('+'))
```

Comment:
How does this code work with other python args? It looks like it could eat any first arg.